src/domain/ecoverse/ecoverse.service.ts

```diff
@@ -192,9 +199,9 @@ export class EcoverseService {
   }
 
   async createGroup(groupName: string): Promise<IUserGroup> {
-    this.logger.verbose(
+    this.logger.verbose?.(
```

Comment:
Why?

Reply:
Because logger was changed to loggerService. NestWinston implements nestJS lLoggerService interface and has verbose as verbose? (optional) and debug as debug? (optional). There is no better way. And I moved to LoggerService because the exception filters require the logger and they are not an injectable service - so a service locator is used on app bootstrap. This approach is strictly following https://github.com/gremo/nest-winston documentation.